justfile

```diff
@@ -0,0 +1,155 @@
+export RUSTFLAGS := "-D warnings"
+export RUSTDOCFLAGS := "-D warnings"
+
+
+
+[private]
+default:
+	just -l --unsorted
+
+
+
+# Install rust 1.65, cargo-deny, cargo-fuzz, cargo-hack, and cargo-semver-checks
+install: install_rust_1_65 \
+         install_cargo_deny \
+	 install_cargo_fuzz \
+	 install_cargo_hack \
+	 install_cargo_semver_checks 
+
+[private]
+install_rust_1_65:
```

Comment:
Why 1.65? Is this the MSRV? If yes, please add a comment.

And would it make sense to use `+1.65` for all cargo commands? E.g. `cargo +1.65`.